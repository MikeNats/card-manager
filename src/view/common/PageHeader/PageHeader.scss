.page-header{ 
  margin-bottom: 56px;

  h1{
    font-size: 24px;
    color: #4c00c2;
    margin: 0 0 16px 0; 
    font-weight: bold;
  }
  p{
    color:$grey-70;
    font-size: 14px;
  }
}  