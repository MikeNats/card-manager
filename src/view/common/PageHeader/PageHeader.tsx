import React from 'react';
import {PageHeaderProps} from './types'
const PageHeader = ({title , description, className}: PageHeaderProps) =>
  <header className="page-header">
    <h1 className={className}>{title}</h1>
    <p>{description}</p>
  </header>
  
 
export default PageHeader 