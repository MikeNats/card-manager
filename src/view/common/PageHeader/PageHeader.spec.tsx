import React from 'react'
import PageHeader from './PageHeader'
import { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import Enzyme from 'enzyme';

Enzyme.configure({ adapter: new Adapter() })

describe("PageHeader", () => {    
    it('renders as expected', () => {
     const component = shallow(
        <PageHeader
          description="description"
          title="title"/>)

      expect(component).toMatchSnapshot();
    })

    it('should set props', () => {	
      const component = shallow(<PageHeader  
        description="description"
        title="title"/>)
        expect(component.find('h1').text()).toEqual('title');
        expect(component.find('p').text()).toEqual('description');

    });

});
