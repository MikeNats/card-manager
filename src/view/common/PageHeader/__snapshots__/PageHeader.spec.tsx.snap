// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageHeader renders as expected 1`] = `ShallowWrapper {}`;
